Models/Enums/Enums.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Models.Enums
+{
+    public abstract class Enums
+    {
+        public enum ServiceCategory : int
+        {
+            Barbershop = 1
+        };
+
+        public enum ServiceSegment : int
+        {
+            Econom = 1,
+            Medium = 2,
+            Premium = 3
+        };
+
+        public enum UserType : int
```

Comment:
these enums should be added in db table
Id [long]
Name [stirng]
Type [int]

Reply:
it's renamed in table as "role", will fix